### DISCUSSION TITLE
Searching tool

### DISCUSSION THREAD

**Discussion by fabrizioschiavi** [ryanoasis/nerd-fonts#1469] [Show and tell]
Hello everyone,

I am pleased to introduce you to an essential tool for searching **nerd font symbols using text**:

[PragmataPro Semiotics](https://fabrizioschiavi.github.io/pragmatapro-semiotics/)

This website allows you to search for over 12,000 symbols by entering keywords in the search field, providing 
Unicode codepoints, UTF-8 symbols, and their corresponding descriptions.

Many symbols still need descriptions, so anyone who wishes to contribute to the [repo](https://github.com/fabrizioschiavi/pragmatapro-semiotics) is welcome.

Thank you for your attention.

**Comment by haunt98** [comment]
Hi @fabrizioschiavi, thanks for great tool.
Just have 1 question, is there a way to replace your PragmataPro font with some font to use with your search tool?

**Reply by fabrizioschiavi** [reply]
While it is true that Nerd Font has become a widely adopted standard among many fonts, this operation may be unnecessary.
